bindings/java/build-unix.xml

```diff
@@ -27,6 +27,42 @@
 		</java>
 	</target>
 
+	<condition property="online">
+		<http url="https://www.google.com/"/>
+	</condition>
+
+	<target name="set-library-path"  description="sets the path of the libraries" depends="set-library-path-online,set-library-path-offline">
+
+	</target>
+
+	<target name="set-library-path-online" description="set this library path when the user is online" if="online">
+		<path id="libraries">
+		<fileset dir="${lib}">
+			<include name="*.jar"/>
+		</fileset>
+		<pathelement path="${build}"/>
+		</path>
+		<echo message="${online}"/>
```

Comment:
Is this needed?

Reply:
I used that for testing I will remove and make another request